lib/private/machines/create-each-record.js

```diff
@@ -105,15 +106,19 @@ module.exports = {
       //  ╠═╝╠╦╝║ ║║  ║╣ ╚═╗╚═╗  │││├─┤ │ │└┐┌┘├┤   ├┬┘├┤ │  │ │├┬┘ │││ └─┐ │
       //  ╩  ╩╚═╚═╝╚═╝╚═╝╚═╝╚═╝  ┘└┘┴ ┴ ┴ ┴ └┘ └─┘  ┴└─└─┘└─┘└─┘┴└──┴┘└─└─┘─┘
       // Process record(s) (mutate in-place) to wash away adapter-specific eccentricities.
```

Comment:
^ This comment makes more sense down below above the try/catch when we process records.

And instead, before that, we need a new comment explaining what we're doing in terms of fetching these records.

Minor thing, but I misunderstood when first reading so others are likely to as well (this'll prevent future confusion)